pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -545,10 +546,26 @@ func newTestAutoscalerWithScalingMetric(t *testing.T, targetValue, targetBurstCa
 		metrics, pc, deciderSpec, ctx), pc
 }
 
+// equateInt32 equates int32s with given path with ±-1 tolerance.
+// This is needed due to rounding errors across various platforms.
+func equateInt32(field string) cmp.Option {
```

Comment:
Maybe equateApproxInt32 or sthg like that?

Reply:
Sure. Don't care much given it's internal to the file.